utils/meeting-utils.js

```diff
@@ -0,0 +1,68 @@
+import { getItemFromLocalStorage } from "./local-storage-utils.js"
+import { getTimeObject } from "./time-utils.js";
+
+/**
+ * Retrieves meeting details from local storage based on the specified room name.
+ * @param roomName - Optional room name to filter the meeting details.
+ * @returns  An array containing meeting details filtered by the specified room name,
+ *                  or all meeting details if no room name is provided, or an empty array if there are no meetings.
+ */
+export const getMeetingDetails = (roomName) => {
+    try {
+        const meetings = JSON.parse(getItemFromLocalStorage('meetings'));
```

Comment:
could you let me know why JSON.parse is used most of the times with getItemFromLocalStorage

Reply:
JSON.parse is used whenever we get the array of objects from the local storage as it is stringified.